src/apiServices/AuthService.ts

```diff
@@ -117,6 +132,15 @@ export class AuthService {
         }
     }
 
+    public async getChallenge(): Promise<any | void> {
```

Comment:
Can we add documentation comments here and at the above function? Essentially copying the comments from `ClientService` is probably appropriate.